test/transmuxer.test.js

```diff
@@ -3408,6 +3408,416 @@ QUnit.test('adjusts caption and ID3 times when configured to adjust timestamps',
   QUnit.equal(captions[0].endTime, (90004 - 90002) / 90e3, 'caption end time are based on original timeline');
 });
 
+QUnit.test('Audio frames trimmed before video, keepOriginalTimestamps = false', function() {
+  var
+    segments = [],
+    segmentLengthOnDone,
+    earliestDts = 15000,
+    transmuxer = new Transmuxer({
+      keepOriginalTimestamps: false
+    });
+
+  transmuxer.on('data', function(segment) {
+    segments.push(segment);
+  });
+  transmuxer.on('done', function(segment) {
+    if (!segmentLengthOnDone) {
```

Comment:
Is this checked in case there are multiple dones?

Reply:
I am just going to remove the `segmentLengthOnDone` code as it is irrelevant to the test.